cli/src/diff_util.rs

```diff
@@ -2029,7 +2051,10 @@ pub fn show_diff_stats(
     let max_diffs = stats
         .entries()
         .iter()
-        .map(|stat| stat.added + stat.removed)
+        .filter_map(|stat| match &stat.stats {
+            DiffStatStats::Lines { added, removed } => Some(added + removed),
+            DiffStatStats::Bytes { .. } => None,
+        })
```

Comment:
Perhaps, we'll need to calculate maximum length of ` | Bin xxx -> yyy bytes` to allocate space for that.

Reply:
Is there a test that exercises these narrow screen widths?  I looked a bit but I am not sure where to look.

Reply:
search `COLUMNS` in test_diff_command.rs